spec/customer_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'customer'
+
+describe Customer do
+  let(:order) { double(:order, :build => true, print: "your order", varify: true) }
+  subject(:customer) { Customer.new("Will", "07539204924", order) }
+  it 'can select a number of available dishes' do
+    expect(customer.select_dishes([1, 2])).to eq(true)
+  end
+
+  it 'can varify order' do
+    customer.select_dishes([1, 2])
+    customer.varify_order
+    expect(customer.print_order).to eq("your order")
```

Comment:
the description says that this test is to do with verification but there's also an expectation about printing.

Reply:
Do you really care what is returned by `print_order` or is it only important that the order receives the right message?

Reply:
Hmm that is a good point - perhaps I was not really thinking about SRP in this test. The expectation about printing is pretty unrelated to the point of the test so should probably go.